test/jdk/javax/swing/JColorChooser/Test4759934.java

```diff
@@ -24,57 +24,78 @@
 /*
  * @test
  * @bug 4759934
+ * * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
  * @summary Tests windows activation problem
- * @author Andrey Pikalev
- * @run applet/manual=yesno Test4759934.html
+ * @run main/manual Test4759934
  */
 
 import java.awt.Color;
 import java.awt.Component;
 import java.awt.Window;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
-import javax.swing.JApplet;
 import javax.swing.JButton;
 import javax.swing.JColorChooser;
 import javax.swing.JDialog;
 import javax.swing.JFrame;
 
-public class Test4759934 extends JApplet implements ActionListener {
+public class Test4759934 {
     private static final String CMD_DIALOG = "Show Dialog"; // NON-NLS: first button
     private static final String CMD_CHOOSER = "Show ColorChooser"; // NON-NLS: second button
 
-    private final JFrame frame = new JFrame("Test"); // NON-NLS: frame title
+    private static JFrame frame;
 
-    public void init() {
-        show(this.frame, CMD_DIALOG);
-    }
+    private static ActionListener al = new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent event) {
+            String command = event.getActionCommand();
+            if (CMD_DIALOG.equals(command)) {
+                JDialog dialog = new JDialog(frame, "Dialog"); // NON-NLS: dialog title
+                dialog.setLocation(200, 0);
+                show(dialog, CMD_CHOOSER, true);
+            }
+            else if (CMD_CHOOSER.equals(command)) {
+                Object source = event.getSource();
+                Component component = (source instanceof Component)
+                        ? (Component) source
+                        : null;
 
-    public void actionPerformed(ActionEvent event) {
-        String command = event.getActionCommand();
-        if (CMD_DIALOG.equals(command)) {
-            JDialog dialog = new JDialog(this.frame, "Dialog"); // NON-NLS: dialog title
-            dialog.setLocation(200, 0);
-            show(dialog, CMD_CHOOSER);
+                JColorChooser.showDialog(component, "ColorChooser", Color.BLUE); // NON-NLS: title
+            }
         }
-        else if (CMD_CHOOSER.equals(command)) {
-            Object source = event.getSource();
-            Component component = (source instanceof Component)
-                    ? (Component) source
-                    : null;
+    };
 
-            JColorChooser.showDialog(component, "ColorChooser", Color.BLUE); // NON-NLS: title
-        }
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4759934")
+                .instructions("Click on show Dialog. Then click on Show ColorChooser.\n" +
+                        "If a ColorChooser is shown, then pass.")
```

Comment:
Why is this just the first part of the instructions as given in the previous HTML ?